src/test/test_ep_cred_client.c

```diff
@@ -235,9 +193,9 @@ test_fini()
 
 	rc = crt_finalize();
 	D_ASSERTF(rc == 0, "crt_finalize() failed. rc: %d\n", rc);
-	D_DEBUG(DB_TRACE, "exiting.\n");
 
 	d_log_fini();
+	D_DEBUG(DB_TRACE, "exiting.\n");
```

Comment:
either this should be DBG_PRINT or moved back before d_log_fini. D_DEBUG becomes a no-op once log has been closed